src/git/zcl_abapgit_git_utils.clas.testclasses.abap

```diff
@@ -1,32 +1,77 @@
 
-CLASS ltcl_length_utf8 DEFINITION FOR TESTING DURATION SHORT RISK LEVEL HARMLESS.
+CLASS ltcl_git_utils DEFINITION FOR TESTING DURATION SHORT RISK LEVEL HARMLESS.
 
   PRIVATE SECTION.
     DATA:
       mo_cut TYPE REF TO zcl_abapgit_git_utils.
 
     METHODS:
       setup,
+      get_null FOR TESTING,
+      pkt_string FOR TESTING RAISING zcx_abapgit_exception,
       length_utf8_hex FOR TESTING RAISING zcx_abapgit_exception.
 ENDCLASS.
 
 
-CLASS ltcl_length_utf8 IMPLEMENTATION.
+CLASS ltcl_git_utils IMPLEMENTATION.
 
   METHOD setup.
     CREATE OBJECT mo_cut.
   ENDMETHOD.
 
+  METHOD get_null.
+
+    CONSTANTS lc_null TYPE x LENGTH 2 VALUE '0000'.
+
+    DATA:
+      lv_c TYPE c LENGTH 1,
+      lv_x TYPE x LENGTH 2.
+
+    lv_c = mo_cut->get_null( ).
+
+    lv_x = lv_c.
```

Comment:
this doesnt test the right thing? moving any non hex value from char to hex will give the initial hex value, 

```abap
DATA lv_c TYPE c LENGTH 1.
DATA lv_x TYPE x LENGTH 2.
lv_c = 'Q'.
lv_x = lv_c.
WRITE lv_x.
```